contracts/oracles/TwapOracle.sol

```diff
@@ -63,6 +56,12 @@ contract TwapOracle is OwnableUpgradeable, TwapInterface {
         uint256 newAcc
     );
 
+    ///@notice Keeps track of observation of token mapped by address , update on evry updateTwap invocation.
```

Comment:
```suggestion
    /// @notice Keeps track of observation of token mapped by address , update on evry updateTwap invocation.
```
I'd assume this would have gotten picked up by linting

Reply:
I also think we should order the entities in this contract per spec